packages/core/src/crypto/jose/jwk/K256Jwk.ts

```diff
@@ -0,0 +1,112 @@
+import type { JwkJson } from './Jwk'
+import type { JwaEncryptionAlgorithm } from '../jwa/alg'
+
+import { TypedArrayEncoder, Buffer } from '../../../utils'
+import { KeyType } from '../../KeyType'
+import { JwaCurve, JwaKeyType } from '../jwa'
+import { JwaSignatureAlgorithm } from '../jwa/alg'
+
+import { Jwk } from './Jwk'
+import { compress, expand } from './ecCompression'
+import { hasKty, hasCrv, hasX, hasY, hasValidUse } from './validate'
+
+export class K256Jwk extends Jwk {
+  public static readonly supportedEncryptionAlgorithms: JwaEncryptionAlgorithm[] = []
+  public static readonly supportedSignatureAlgorithms: JwaSignatureAlgorithm[] = [JwaSignatureAlgorithm.ES256]
```

Comment:
This should be `ES256K` right?

```suggestion
  public static readonly supportedSignatureAlgorithms: JwaSignatureAlgorithm[] = [JwaSignatureAlgorithm. ES256K]
```

Reply:
Good catch :)